src/ert/ensemble_evaluator/_builder/_legacy.py

```diff
@@ -31,6 +41,11 @@
 event_logger = logging.getLogger("ert.event_log")
 
 
+class _JobQueue(Protocol):
```

Comment:
Oh no, yet another usage of "Job"... 😢 https://github.com/equinor/ert/issues/6779

Reply:
It's a `Protocol` class for `JobQueue`. I don't believe I need it though.

Reply:
Renamed to `_KillAllJobs` since that is the only thing it does.